Foundation HTML, CSS, and Git improvements (#88)
HackYourFuture-CPH/programme: courses/Foundation/git/assignment.md

- `git branch <branch-name>` - to create a new branch named `<branch-name>`
- `git checkout <branch-name>` - to move to a branch named `<branch-name>`
- `git add <file_name>` - tell git to start tracking a file and to update what will be committed

> - `git add .` - tell git to add to staging for commit all modified files

Reviewer: I'm not sure I have a better suggestion, but the wording of that line feels clumsy.

Maybe:

```suggestion
- `git add .` - add all files in this directory, to what will be committed
```
